deployments/forgeops/overlay/config/7.0/cdk/idm/conf/access.json

```diff
@@ -280,13 +286,6 @@
       "methods"   : "action",
       "actions"   : "deleteNotificationsForTarget",
       "customAuthz" : "request.additionalParameters.target === (context.security.authorization.component + '/' + context.security.authorization.id)"
-    },
-    {
-      "pattern"   : "managed/*",
```

Comment:
Just out of curiosity, why was this removed?

Reply:
Not sure, but the details are here if you want to look into it: https://github.com/ForgeRock/forgeops/commit/fbadcd627738e01b3db40da2a3096bc72fe24afd#diff-1f6d6e19f23d7e0a0824a4d73911d6f57b430727d7400044bb8920284f281661